proto/v3/message_contents/invite.proto

```diff
@@ -0,0 +1,39 @@
+// V3 invite message structure
+syntax = "proto3";
+
+package xmtp.v3.message_contents;
+
+import "v3/message_contents/public_key.proto";
+
+option go_package = "github.com/xmtp/proto/v3/go/v3/message_contents";
+option java_package = "org.xmtp.proto.v3.message.contents";
+
+// InvitationV1 is the encrypted invitation message meant to be encrypted as
+// ciphertext in SealedInvitationV1 and decrypted by the recipient using the
+// provided inviter `InstallationContactBundle`
+message InvitationV1 {
+    // The inviter_wallet_address must match the address derived from the
+    // signature in the inviter contact bundle. Either the
+    // `inviter_wallet_address` or the `recipient_wallet_address` MUST match the
+    // wallet address of the account receiving this invite
+    string inviter_wallet_address = 1;
+    string recipient_wallet_address = 2;
+    string conversation_unique_id = 3;
+    // TODO: Decide whether we need a Context field
+}
```

Comment:
The address fields specifically refer to wallet addresses. How does this work in a pairwise channel between two installations -- which is needed for forward messages from one users client to the other?

Do Invitations logic only exist at the account level? is there any identifier for pairwise channels?

Reply:
Once you decrypt the invitation with vmac, you will automatically get a `session_id`. I'm implicitly proposing we use that `session_id` as the identifier for the pairwise channel.